package/sap-installation-wizard.spec

```diff
@@ -19,7 +19,7 @@ Name:           sap-installation-wizard
 Summary:        Installation wizard for SAP applications
 License:        GPL-2.0+
 Group:          System/YaST
-Version:        4.4.9
+Version:        4.4.12
```

Comment:
Why jumping 3 versions?

Reply:
We have developed SAPBone for SLES-SP4. This have be made in the "hot phase" very rapidly and I had to reackt very quickly. As result 4.4.11 is already released in SUSE:SLE-15-SP4:Update:Products:SAPBOne:Update/sap-installation-wizard.